cocktails.mcp/src/.env

```
@@ -1,9 +1,8 @@
 APPLICATIONINSIGHTS_INSTRUMENTATIONKEY=
-AZUREAD_B2C_CLIENT_ID=
-AZUREAD_B2C_DOMAIN=
-AZUREAD_B2C_INSTANCE=
-AZUREAD_B2C_USERFLOW=
+AUTH0_CLIENT_ID=
+AUTH0_DOMAIN=
+AUTH0_AUDIENCE=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUTH0_AUDIENCE key should go before the AUTH0_CLIENT_ID key

(UnorderedKey)